_events/2020/2020-april-community-workshop.md

```diff
@@ -0,0 +1,26 @@
+---
+title: US-RSE Community Building Workshop
```

Comment:
The content looks good, and links check out! My one critique is that the title is a bit chonky:

![image](https://user-images.githubusercontent.com/814322/69974256-1e325180-14e2-11ea-81a2-d8d10024f2cb.png)

What about changing the title to "US-RSE Workshop" and then adding the rest as an h3 or h4 below it? For example:

![image](https://user-images.githubusercontent.com/814322/69974410-6ea9af00-14e2-11ea-95f5-e4461740b103.png)

Or if you think it's an issue with the overall events page styling (h1 is not appropriate) we could try adjusting that too.

Reply:
I agree it looks clunky, and probably the overall events page would look better with the ability to have a longer event name.  But for the purpose of this, I'll make the change to a short one with a subtitle.  It'll be fine.  Let's see if we have an issue again, and then consider changing the underlying layout.